pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/KitUploadRoute.java

```diff
@@ -404,8 +403,15 @@ private void addKitRequest(Connection conn, String kitTypeName, KitRequestSettin
 
             //If there is a participant change the participantID to the ID of the existing
             //participant
-            if (StringUtils.isNotBlank((ddpInstance.getParticipantIndexES()))) {
-                participantID = kit.getParticipantId().trim();
+            if (ddpInstance.hasEsIndex()) {
```

Comment:
How about ddpInstance.migratedDDP instead of the index check?

Reply:
Same as the other comment, TBH I don't think we can trust that field

Reply:
Thanks Zim for explaining this to me in the messages. I am working on dev and test db to reflect prod